services/self-service/src/main/java/com/epam/dlab/backendapi/dao/EndpointDAOImpl.java

```diff
@@ -65,11 +74,23 @@ public void remove(String name) {
 		deleteOne(ENDPOINTS_COLLECTION, endpointCondition(name));
 	}
 
+	/**
+	 * According the functional requirement, the EndPoint name should be case-insensitive.
+	 * A New method use the regex interface of the MongoDB driver
+	 */
+	@Deprecated //private Bson endpointCondition(String name) { return eq(ENDPOINT_NAME_FIELD, name); }
```

Comment:
Why do you use '@Deprecated' here?
I think we should not leave the old code in our code base

Reply:
The depricated method has been removed